stwcs/updatewcs/__init__.py

```diff
@@ -93,6 +102,11 @@ def updatewcs(input, vacorr=True, tddcorr=True, npolcorr=True, d2imcorr=True,
             print('No valid input, quitting ...\n')
             return
 
+    if use_db:
+        # Establish any available connection to
+        #  an accessible astrometry web-service
+        astrometry = astrometry_utils.AstrometryDB()
```

Comment:
`use_db` is true by default so calling `updatewcs` results always in an attempt to create a connection. You need to handle the case when a connection is not available so it doesn't crash. This is mostly why the tests are currently failing.

Reply:
It turned out that there were problems with the logic that was used to interpret when to raise exceptions and when to perform the astrometry update at all.  Those errors were resolved and the tests for altwcs and updatewcs were updated to set the environment variable to turn off applying the astrometry update at all.  This now allows the tests to pass, and still provide the control needed by the pipeline for using this new code.  Please let me know if you feel there are any additional problems with this code that needs to be addressed.